src/Splunk.Client/Splunk/Client/JobCollection.cs

```diff
@@ -195,6 +195,31 @@ public override async Task<Job> CreateAsync(IEnumerable<Argument> arguments)
             return job;
         }
 
+        public virtual async Task<Job> Create(string search, int count = 0, // This version doesn't wait
```

Comment:
We can remove these, but I wanted to add another `Create()` function that doesn't wait until a `requiredStatus` is reached